addons/mail/static/src/core/common/message_model.js

```diff
@@ -37,6 +35,8 @@ export class Message extends Record {
     author = Record.one("Persona");
     body = Record.attr("", { html: true });
     composer = Record.one("Composer", { inverse: "message", onDelete: (r) => r.delete() });
+    /** @type {DateTime} */
```

Comment:
no way to have automatic type from the type param?

Reply:
I don't see one, sadly